### PR TITLE
Michaelw/recommended metrics

### PR DESCRIPTION
**Pull request by mstbbs**
### What does this PR do?
Add documentation for recommended usage metrics

https://docs-staging.datadoghq.com/michaelw/recommended-metrics/logs/logs_to_metrics/#recommended-usage-metrics

https://docs-staging.datadoghq.com/michaelw/recommended-metrics/account_management/billing/usage_metrics

### CODE REVIEW

File: content/en/logs/logs_to_metrics.md [hunk 1]
```diff
@@ -46,9 +46,21 @@ After a metric is created, only these fields can be updated:
...
+{{< img src="logs/processing/logs_to_metrics/recommended_usage_metrics.png" alt="Recommended Usage Metrics" responsive="true" style="width:80%;">}}
+
+Usage metrics are estimates of your current Datadog usage in near real-time. They enable you to:
+
+* Graph your estimated usage
+* Create monitors around your estimated usage
+* Get instant alerts of spikes or drops in your usage
+* Assess the potential impact of code changes on your usage in near real-time
+
+Log Management usage metrics come with three dimensions that can be used for more granular monitoring:
```
Comment (l0k0ms) [inline content/en/logs/logs_to_metrics.md]:
```suggestion
Log Management usage metrics come with three dimensions that can be used for more granular monitoring:

```

File: content/en/logs/logs_to_metrics.md [hunk 2]
```diff
@@ -46,9 +46,21 @@ After a metric is created, only these fields can be updated:
...
 
-## Feedback
+## Recommended usage metrics
 
-Datadog would love to receive your feedback on this feature. You can submit your feedback directly from [this form][9].
+{{< img src="logs/processing/logs_to_metrics/recommended_usage_metrics.png" alt="Recommended Usage Metrics" responsive="true" style="width:80%;">}}
+
+Usage metrics are estimates of your current Datadog usage in near real-time. They enable you to:
+
+* Graph your estimated usage
```
Comment (l0k0ms) [inline content/en/logs/logs_to_metrics.md]:
```suggestion
* Graph your estimated usage.
```

File: content/en/logs/logs_to_metrics.md [hunk 3]
```diff
@@ -46,9 +46,21 @@ After a metric is created, only these fields can be updated:
...
+* Create monitors around your estimated usage
```
Comment (l0k0ms) [inline content/en/logs/logs_to_metrics.md]:
```suggestion
* Create monitors around your estimated usage.
```

File: content/en/logs/logs_to_metrics.md [hunk 4]
```diff
@@ -46,9 +46,21 @@ After a metric is created, only these fields can be updated:
...
+* Get instant alerts of spikes or drops in your usage
```
Comment (l0k0ms) [inline content/en/logs/logs_to_metrics.md]:
```suggestion
* Get instant alerts of spikes or drops in your usage.
```

File: content/en/logs/logs_to_metrics.md [hunk 5]
```diff
@@ -46,9 +46,21 @@ After a metric is created, only these fields can be updated:
...
+* Assess the potential impact of code changes on your usage in near real-time
```
Comment (l0k0ms) [inline content/en/logs/logs_to_metrics.md]:
```suggestion
* Assess the potential impact of code changes on your usage in near real-time.
```

File: content/en/logs/logs_to_metrics.md [hunk 6]
```diff
@@ -46,9 +46,21 @@ After a metric is created, only these fields can be updated:
...
+
+Log Management usage metrics come with three dimensions that can be used for more granular monitoring:
+- `datadog_index` indicates the routing query that matches a log to an intended index
```
Comment (l0k0ms) [inline content/en/logs/logs_to_metrics.md]: Could we put example here of tags that are available?

Comment (NBParis) [reply content/en/logs/logs_to_metrics.md]: Not sure I understand the question, there are 3 tags by default which are the listed one. Are you talking about custom ones?

Comment (l0k0ms) [reply content/en/logs/logs_to_metrics.md]: it was something like `datadog_index:main`, `datadog_index:<CUSTOM_INDEX>`..

**Pull request merged**